tests/client/test_job.py

```diff
@@ -44,6 +45,49 @@ def test_job_update_tags(spider):
     assert job2.metadata.get('tags') == ['tag2']
 
 
+def test_cancel_jobs(spider):
```

Comment:
Looks good, could you please split the large single test to a few simpler ones, and recreate cassettes? The small tests would help with getting a whole picture when something is broken.

Reply:
Good point, I'm going to change it :)